packages/edit-widgets/src/components/widget-areas-block-editor-content/index.js

```diff
@@ -17,31 +17,27 @@ import Notices from '../notices';
 import KeyboardShortcuts from '../keyboard-shortcuts';
 
 export default function WidgetAreasBlockEditorContent() {
-	const { clearSelectedBlock } = useDispatch( 'core/block-editor' );
-
 	return (
-		<>
-			<KeyboardShortcuts />
-			<BlockEditorKeyboardShortcuts />
-			<Notices />
-			<Popover.Slot name="block-toolbar" />
-			<div tabIndex="-1" onFocus={ clearSelectedBlock }>
-				<div
-					className="editor-styles-wrapper"
-					onFocus={ ( event ) => {
-						// Stop propagation of the focus event to avoid the parent
-						// widget layout component catching the event and removing the selected area.
-						event.stopPropagation();
-						event.preventDefault();
-					} }
-				>
-					<WritingFlow>
-						<ObserveTyping>
-							<BlockList className="edit-widgets-main-block-list" />
-						</ObserveTyping>
-					</WritingFlow>
-				</div>
+		<BlockSelectionClearer>
```

Comment:
I also replaced the original `<div>` with `<BlockSelectionClearer>`. Not sure why we were not using it in the first place?

Reply:
Probably just something missed at some point.